variables.tf

```diff
@@ -151,6 +151,26 @@ variable "se_instance_type" {
   type        = string
   default     = "c5.large"
 }
+variable "se_s3_encryption" {
+  description = "Enable encryption on SE S3 Bucket"
+  type        = bool
+  default     = "false"
+}
+variable "se_ebs_encryption" {
+  description = "Enable encryption on SE S3 Bucket"
```

Comment:
looks like this needs to be updated to reflect the SE volume/ami instead

Reply:
Good catch.  I'll fix that